session-replacement/src/main/java/com/amadeus/session/repository/redis/RedisConfiguration.java

```diff
@@ -205,7 +247,13 @@ private void parseArgFromConfiguration(String arg) {
       clusterMode = arg.substring(CLUSTER_MODE_PROPERTY.length());
     } else if (arg.startsWith(HOST_PROPERTY)) {
       server = arg.substring(HOST_PROPERTY.length());
-    } else if (arg.startsWith(MASTER_NAME_PROPERTY)) {
+    } else if (arg.startsWith(PASSWORD_PROPERTY)){
+      password = arg.substring(PASSWORD_PROPERTY.length());
+    } else if (arg.startsWith(SSL_PROPERTY)){
+      useSSL = stringToBool(arg.substring(SSL_PROPERTY.length()));
```

Comment:
Boolean.parseString(str.trim()) has same effect.

Reply:
Initially.. I had that as `Boolean.parseBoolean(arg.substring(SSL_PROPERTY.length()).trim())` but for something reason it wasn't working.  I'll try it out again.